devices/ePaper/Fonts/Font8x12.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright (c) 2022 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+
+namespace Iot.Device.ePaper.Fonts
+{
+    /// <summary>
+    /// A font class with every character is 12-points wide and 8-points high.
+    /// </summary>
+    public class Font8x12 : IFont
```

Comment:
Note for later: have nugets with different fonts ready to use with IFont (we do already have few of those a bit everywhere). Not for this PR!